### DISCUSSION TITLE
"Pre-release" is in front of "Latest release"

### DISCUSSION THREAD

**Discussion by plainheart** [community/community#6409] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, the new UI design for Releases is better and better. I just noticed that an early "Pre-release" is in front of the "Latest release"(https://github.com/apache/echarts/releases). Is it expected behavior?

**Comment by MylesBorins** [comment, marked as answer]
@plainheart this is definitely an interesting sorting / order issue. We'll dig in and see what we can do to improve it!

**Reply by MylesBorins** [reply]
Our fix has landed PTAL

**Reply by plainheart** [reply]
Thanks for your work! The order looks good now.
